_inc/client/plans/single-product-backup.scss

```diff
@@ -1,3 +1,15 @@
+.plans-section__body {
+	h4 {
+		font-size: 14px;
+		color: #747474;
+		border-bottom: solid 2px #e6e6e6;
```

Comment:
I think that the figma design says the border-width should be 1px.

Reply:
I'm pretty sure it's 2px, I checked in the Figma both by zooming in to count pixels and looking at the property value.

Reply:
Maybe we're using different Figma files for reference? Here's a screenshot of what I can see in the "Backups Single Product Only" project:

![Screenshot 2019-11-15 at 10 54 00](https://user-images.githubusercontent.com/478735/68934088-5160ab00-0796-11ea-9694-4d5581fdae4f.png)

In any case, we can leave it as is. In case the designers want to change it to 1px, we can do it easily later on.